nexus/types/src/external_api/views.rs

```diff
@@ -1585,6 +1585,25 @@ fn expected_one_of<T: strum::VariantArray + fmt::Display>() -> String {
     msg
 }
 
+// SCIM
+
+/// The POST response is the only time the generated bearer token is returned to
+/// the client.
+#[derive(Deserialize, Serialize, JsonSchema)]
+pub struct ScimClientBearerTokenCreateResponse {
```

Comment:
We have a couple of other view structs that end in `Response`, but I prefer to avoid it if possible. We might be stuck with it, though. Just taking out "response" doesn't work because our create params all end in `Create`.